src/main/java/seedu/address/model/person/ReadOnlyPerson.java

```diff
@@ -12,6 +13,19 @@
  */
 public interface ReadOnlyPerson {
 
+    Comparator<ReadOnlyPerson> NAMESORTASC = (ReadOnlyPerson o1, ReadOnlyPerson o2)
+        -> o1.getName().compareTo(o2.getName());
+    Comparator<ReadOnlyPerson> PHONESORTASC = (ReadOnlyPerson o1, ReadOnlyPerson o2)
+        -> o1.getPhone().compareTo(o2.getPhone());
+    Comparator<ReadOnlyPerson> EMAILSORTASC = (ReadOnlyPerson o1, ReadOnlyPerson o2)
+        -> o1.getEmail().compareTo(o2.getEmail());
+    Comparator<ReadOnlyPerson> NAMESORTDSC = (ReadOnlyPerson o1, ReadOnlyPerson o2)
+        -> o2.getName().compareTo(o1.getName());
+    Comparator<ReadOnlyPerson> PHONESORTDSC = (ReadOnlyPerson o1, ReadOnlyPerson o2)
+        -> o2.getPhone().compareTo(o1.getPhone());
+    Comparator<ReadOnlyPerson> EMAILSORTDSC = (ReadOnlyPerson o1, ReadOnlyPerson o2)
+        -> o2.getEmail().compareTo(o1.getEmail());
```

Comment:
These comparators can be put elsewhere in my opinion, this is probably not the best place to put them.
Recall Single Responsibility Principle.

Reply:
In fact, I think the comparator object can be built on demand, only when `sort` command is called, instead of being stored as fields.